hpobench/container/client_abstract_benchmark.py

```diff
@@ -62,14 +62,15 @@ class AbstractBenchmarkClient(metaclass=abc.ABCMeta):
     socket_id : str
     """
     def __init__(self, benchmark_name: str, container_name: str, container_source: Optional[str] = None,
-                 container_tag: str = 'latest', gpu: Optional[bool] = False,
-                 rng: Union[np.random.RandomState, int, None] = None, **kwargs):
+                 container_tag: str = 'latest', env_str: Optional[str] = '', bind_str: Optional[str] = '',
+                 gpu: Optional[bool] = False, rng: Union[np.random.RandomState, int, None] = None, **kwargs):
 
         self.socket_id = self._id_generator()
-        self._setup(benchmark_name, container_name, container_tag, container_source, gpu, rng, **kwargs)
+        self._setup(benchmark_name, container_name, container_tag, container_source, env_str, bind_str, gpu, rng,
+                    **kwargs)
 
     def _setup(self, benchmark_name: str, container_name: str, container_tag: str,
-               container_source: Optional[str] = None,
+               container_source: Optional[str] = None, env_str: Optional[str] = '', bind_str: Optional[str] = '',
```

Comment:
Since args for `__init__()` and `_setup()` are the same, could you also order them in the same way?